content/sensu-go/5.5/sensuctl/reference.md

```diff
@@ -228,7 +228,7 @@ cat my-resources.json | sensuctl create
 `HookConfig` | `hook_config` | `Mutator` | `mutator`
 `Namespace` | `namespace` | `Role` | `role`
 `RoleBinding` | `role_binding` | `Silenced` | `silenced`
-[`ldap`][26] | [`ad`][26] | |
+[`ldap`][26] | [`ad`][26] | [`TessenConfig`][27] |
```

Comment:
1. Is there a `sensuctl edit` resource type for Tessen config?
2. What is the RBAC resource type for Tessen config?

Reply:
- `TessenConfig` is not supported with `sensuctl edit` because the command requires that the resource has a name.
- `TessenConfig` can only be modified by the admin user.